lib/public/components/Filters/common/filters/FloatComparisonFilterModel.js

```diff
@@ -0,0 +1,97 @@
+/**
+ * @license
+ * Copyright CERN and copyright holders of ALICE O2. This software is
+ * distributed under the terms of the GNU General Public License v3 (GPL
+ * Version 3), copied verbatim in the file "COPYING".
+ *
+ * See http://alice-o2.web.cern.ch/license for full licensing information.
+ *
+ * In applying this license CERN does not waive the privileges and immunities
+ * granted to it by virtue of its status as an Intergovernmental Organization
+ * or submit itself to any jurisdiction.
+ */
+import { ComparisonSelectionModel } from './ComparisonSelectionModel.js';
+import { FilterInputModel } from './FilterInputModel.js';
+import { Observable } from '/js/src/index.js';
+
+/**
+ * FloatComparisonFilterModel
+ */
+export class FloatComparisonFilterModel extends Observable {
```

Comment:
I would't say that `Amount` is more generic than `Float`, `Amount` implies rather non-negative value, but filtering with negative values is not required for now, then it can be `Amount`, also `NumericalComparisonFilterModel`, or even `ComparisonFilterModel`, because strings would also work.